scrapi/processing/cassandra.py

```diff
@@ -77,8 +160,85 @@ def different(self, old, new):
         except Exception:
             return True  # If the document fails to load/compare for some reason, accept a new version
 
+    def documents(self, *sources):
+        sources = sources
+        q = DocumentModel.objects.timeout(500).allow_filtering().all().limit(1000)
+        querysets = (q.filter(source=source) for source in sources) if sources else [q]
+        for query in querysets:
+            page = try_n_times(5, list, query)
+            while len(page) > 0:
+                for doc in page:
+                    yield DocumentTuple(self.to_raw(doc), self.to_normalized(doc))
+                page = try_n_times(5, self.next_page, query, page)
+
+    def next_page(self, query, page):
+        return list(query.filter(docID__gt=page[-1].docID))
+
+    def to_raw(self, doc):
+
+        doc_items = dict(doc)
```

Comment:
Probably don't need to convert it to a dict, I think all the items should be reachable with just
```python
doc.doc
doc.docID
doc.source
doc.filetype
```
etc